docs/articles/documentation/extending-testcafe/custom-reporter/index.md

```diff
@@ -38,13 +38,12 @@ mkdir my-reporter
 cd my-reporter
 ```
 
-It is recommended to name the directory the same as you would like to name your reporter project. When you run the generator,
-it will automatically suggest the reporter name that matches the reporter directory name. As for the reporter package name,
-it will consist of two parts - the `{testcafe-reporter-}` prefix and the name of the reporter itself:
-
-```text
-{testcafe-reporter-my-reporter}
-```
+> It is recommended to name the directory the same as you would like to name your reporter project. When you run the generator,
+it will automatically suggest the reporter name that matches the reporter directory name.
+>
+> The generator will also automatically create the reporter package name that consists of two parts - the `testcafe-reporter-` prefix and the name of the reporter itself, for example,`testcafe-reporter-my-reporter`.
+>
+> If you name the reporter package manually, it is important that its name starts with the `testcafe-reporter-` prefix. Otherwise, TestCafe will be unable to recognize the plugin.
```

Comment:
This part seems to be far more important than the previous paragraphs, but it gets lost here.